src/app/components/directions/directions.component.ts

```diff
@@ -79,17 +87,27 @@ export class DirectionsComponent{
       });
       
     }
+
+    //Set the destination as the address of the poi
+    this.setDestination(this.address)
   }
 
-  
+  ngOnInit() 
+  {    
+  }  
+
+  //Sets the destination address
+  setDestination(dest: String)
```

Comment:
This is fine as long as the storage module's .ready() function has already been executed. Just a heads up if you see bugs related to this function